### ISSUE TITLE
"unbind" callback fails after removing all listeners

### ISSUE THREAD

**Issue by mnasyrov** [ai/nanoevents#51]
Hi! `unbind` callback fails after removing all listeners by the recommended way by the docs.

Tested on `nanoevents@5.1.8`:

```js
import { createNanoEvents } from 'nanoevents';

const emitter = createNanoEvents();
const unbind = emitter.on('foo', () => undefined);

emitter.events = {};

// Error message: "Cannot read property 'filter' of undefined"
expect(() => unbind()).not.toThrowError();
```

**Comment by ai** [comment]
All listeners removing is just for tests.

What is your use case here?

**Comment by mnasyrov** [comment]
I'm disposing a some resource and need to clean up its emitter completely to make sure listeners will be collected by GC. However, some unbind callbacks may be called lately.

**Comment by ai** [comment]
If you will be able to fix the problem in a few bytes, we can change the logic

**Comment by mnasyrov** [comment]
I think it is enough to add a fallback for "listeners" array before filtration:

```js
on (event, cb) {
    ;(this.events[event] = this.events[event] || []).push(cb)
    return () => (this.events[event] = (this.events[event] || []).filter(i => i !== cb))
}
```

**Comment by ai** [comment]
Send PR. Let’s see the Size Limit difference.

**Comment by mnasyrov** [comment]
Opened PR #52

Personally I think that using size-limit tool is not fun, because we are competing with zip compression algorithm, manual minification skills are useless here.

**Comment by ai** [comment]
The fix was released in 5.1.11.
